package.json

```diff
@@ -47,10 +47,10 @@
     "ws": "2.3.1"
   },
   "scripts": {
-    "build": "babel src --out-dir .",
+    "build": "babel src --out-dir . --source-maps",
```

Comment:
Do we want them for the regular build? Wondering if there's a benefit to including them

Reply:
Yes, the reason to include it in regular build is for get better sentry (or sentry-like) reports (but of course, its not necesary).

Reply:
please don't do this, you could create another script if you want, but this will cause the RN app to have more than it actually needs when building for release

Reply:
Completely agree, was a proposal based on the comments in the redux channel, but of course, is better to remove it :)